plugins/importexport/datacite/locale/pt_PT/common.xml

```diff
@@ -2,55 +2,67 @@
 <!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
 
 <!--
-  * @file plugins/importexport/medra/locale/pt_PT/common.xml
+  * plugins/importexport/datacite/locale/pt_PT/common.xml
```

Comment:
this is the correct file location...I will make a note to change this in the other files of the datacite plugin